sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Conditions/ComparisonCondition.cs

```diff
@@ -18,26 +18,31 @@ internal class ComparisonCondition : ConditionBase
         public string Field { get; set; }
 
         /// <summary>
-        /// The comparison operator being invoked.
-        /// </summary>
-        public string Operator { get; set; }
-
-        /// <summary>
-        /// The value we're checking against a Field. E.g., in the above example, 5 is the value.
+        /// The value we're checking against a Field.
         /// </summary>
         public string Value { get; set; }
```

Comment:
is your value always a String? what user is checking an integer property?

Reply:
ATM, the user would have to enter the number as a string, so `"50"` instead of `50`. I could look into ways to make the type non-specific such that a user could enter an integer/float/etc. as that could be more intuitive.

Reply:
I think the value should either be a native type, or more likely it is a string and the user has to represent it as they would in SQL. For instance, the value could be `5` or it could be `'Nick P'`.